custom/icds/messaging/indicators.py

```diff
@@ -115,8 +140,6 @@ class LSSubmissionPerformanceIndicator(LSIndicator):
     def __init__(self, domain, user):
         super(LSSubmissionPerformanceIndicator, self).__init__(domain, user)
 
-        child_locations = [l.location_id for l in self.user.sql_location.child_locations()]
-        self.aww_user_ids = user_ids_at_locations(child_locations)
```

Comment:
i see you already added primary location to es, that's fine for now, and fine to table this suggestion